Tile for Email Routing (#2626)
cloudflare/cloudflare-docs: products/email-routing/src/content/enable-email-routing.md

---
order: 1
pcx-content-type: how-to
---

# Enable Email Routing

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account and domain. Go to **Email**.
1. If this is your first time opening the Email app, you will be presented with an explanation of what Email Routing is. Click **Configure Email Routing** to start.
1. In **Custom addresses**, click **Create address**. Enter the email alias you want to use in the **Custom address** field (for example, `my-new-email`).
1. In **Destination address**, enter the full email address you want your emails to be forwarded to — for example, `your-name@gmail.com`.
1. Click **Save**.

Cloudflare will send you a verification email to the address provided in the Destination address field. You must click **Verify email address** in the verification email address to activate Email Routing.

## Confirm DNS records

> After creating your email alias and verifying your destination address, Cloudflare needs to add the relevant  MX and TXT records to DNS records for Email Routing to work. This step is automatic and is only needed the first time you configure Email Routing. It is meant to ensure you have the proper records configured in your zone.

Reviewer: ```suggestion
After creating your email alias and verifying your destination address, Cloudflare needs to add the relevant MX and TXT records to DNS records for Email Routing to work. This step is automatic and is only needed the first time you configure Email Routing. It is meant to ensure you have the proper records configured in your zone.
```

---

> title: Getting Started

Reviewer 2: ```suggestion
title: Get started
```

order: 1
pcx-content-type: how-to
---

# Enable Email Routing

> 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account and domain. Go to **Email**.

Reviewer 2: ```suggestion
1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account and domain.
1. Go to **Email**.
```

1. If this is your first time opening the Email app, you will be presented with an explanation of what Email Routing is. Click **Configure Email Routing** to start.

> 1. In **Custom addresses**, click **Create address**. Enter the email address you want to use in the **Custom address** field (for example, `my-new-email`).

Reviewer 2: ```suggestion
1. In **Custom addresses**, click **Create address**. Enter the email address you want to use in **Custom address** (for example, `my-new-email`).
```

1. In **Destination address**, enter the full email address you want your emails to be forwarded to — for example, `your-name@gmail.com`.
1. Click **Save**.

> Cloudflare will send you a verification email to the address provided in the Destination address field. You must click **Verify email address** in the verification email to activate Email Routing.

Reviewer 2: ```suggestion
Cloudflare will send a verification email to the address provided in the Destination address field. You must click **Verify email address** in the verification email to activate Email Routing.
```

## Confirm DNS records

> After creating your email alias and verifying your destination address, Cloudflare needs to add the relevant MX and TXT records to DNS records for Email Routing to work. This step is automatic and is only needed the first time you configure Email Routing. It is meant to ensure you have the proper records configured in your zone.

Reviewer 2: I don't think we called it "alias" before. Should it be "email custom address" also?

> 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account and domain. Go to **Email**. 

Reviewer 2: ```suggestion
1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account and domain.
1. Go to **Email**.
```

1. If this is your first time configuring Email Routing, click **Add records automatically** in the **Email DNS records** card.

<Aside type="note">

> When Email Routing is configured and running, no other email services can be active for this domain. If other MX records are already configured in DNS, the user is offered the opportunity to delete them. Not deleting existing MX records means that Email Routing will not be enabled.

Reviewer 2: ```suggestion
When Email Routing is configured and running, no other email services can be active in the domain you are configuring. If there are other MX records already configured in DNS, Cloudflare will ask you if you wish to delete them. If you do not delete existing MX records, Email Routing will not be enabled.
```
